platform-hub-web/src/app/projects/projects-list.html

```diff
@@ -17,14 +17,16 @@ <h3>
 
   <loading-indicator loading="$ctrl.loading"></loading-indicator>
 
-  <md-content>
+  <md-content id="projects">
 
-    <md-card ng-repeat="p in $ctrl.Projects.all track by p.id">
+    <md-card id="projects" ng-repeat="p in $ctrl.Projects.all track by p.id"
```

Comment:
has same id as `<md-content>` above - id should be unique or use class

Reply:
Resolved - extra id removed from `<md-content>`